src/python/pants/backend/python/subsystems/pytest.py

```diff
@@ -63,7 +63,7 @@ class PyTest(PythonToolBase):
     # This should be kept in sync with `requirements.txt`.
```

Comment:
Thank you - could you also please update the comment here to say:

> This should be compatible with `requirements.txt`, although it can be more precise.